src/components/Navbar/Mobile/MobileCollapse.tsx

```diff
@@ -86,15 +86,17 @@ export default function MobileCollapse() {
 
             <ResourcesDropdownMobile />
 
-            <Menu.Item key="language-selector">
-              <NavLanguageSelector mobile />
-            </Menu.Item>
-            <Menu.Item key="theme-picker">
-              <ThemePickerMobile />
-            </Menu.Item>
-            <Menu.Item key="feedback">
-              <FeedbackFormButton mobile />
-            </Menu.Item>
+            <div className="top-left-nav" style={{ marginLeft: 15 }}>
```

Comment:
**Question: ** any reason we need the className here? I think I remember removing the CSS for this one, don't think it should be in any JSX